alacritty.yml

```diff
@@ -180,6 +180,10 @@
 #draw_bold_text_with_bright_colors: false
 
 # Colors (Tomorrow Night Bright)
+#
+# Color's value could be specified either with '0xrrggbb' or with '#rrggbb'
+# notation.
```

Comment:
```suggestion
# Colors can be specified using either '0xrrggbb' or '#rrggbb` notation.
```

Reply:
Not sure about dropping notation, but I was picking between `Colors` and `Color's value` originally, due to use of `colors` outside of `colors` section, so decided to avoid confusion, however the comment is above colors section, so I feel like it should be fine to use your suggestion here.